stat/sampleuv/weighted_test.go

```diff
@@ -266,3 +266,48 @@ func TestWeightedNoResample(t *testing.T) {
 		}
 	}
 }
+
+func calculateNonZeroWeights(weights []float64) int {
```

Comment:
Please move this to below `FuzzWeightedTake`.

Reply:
Fixed